zfs/replicate/cli/main.py

```diff
@@ -0,0 +1,103 @@
+"""Main function zfs-replicate."""
+
+import itertools
+
+import click
+
+from .. import filesystem, snapshot, ssh, task
+from ..compress import Compression
+from ..filesystem import FileSystem
+from ..filesystem import filesystem as filesystem_t
+from ..ssh import Cipher
+from .click import EnumChoice
+
+
+@click.command()
+@click.option("--verbose", "-v", is_flag=True, help="Print additional output.")
+@click.option("--dry-run", is_flag=True, help="Generate replication tasks but do not execute them.")
+@click.option(
+    "--follow-delete", is_flag=True, help="Delete snapshots on REMOTE_FS that have been deleted from LOCAL_FS."
+)
+@click.option("--recursive", is_flag=True, help="Recursively replicate snapshots.")
```

Comment:
having a shorthand `-r` would be nice here

Reply:
It sure would, but you know this is a release.  Can you open a new issue with that request?

Reply:
https://github.com/alunduil/zfs-replicate/issues/11